test/wasm2js/float_literals-modified.2asm.js.opt

```diff
@@ -38,75 +42,93 @@ function asmFunc(global, env, buffer) {
  var setTempRet0 = env.setTempRet0;
  var i64toi32_i32$HIGH_BITS = 0;
  function $0() {
-  return 2143289344;
+  wasm2js_scratch_store_f32(Math_fround(Math_fround(nan)));
+  return wasm2js_scratch_load_i32(0) | 0;
```

Comment:
Why has values in this file changed to a load and a store? Why wasn't this load/store before?

Reply:
Good point, this does look wrong. It might be that AvoidReinterprets no longer is able to optimize this out, which means we need to improve it for this... I'll look into that.